Cargo.toml

```diff
@@ -280,6 +280,7 @@ async-recursion = "1.0.0"
 async-tar = "0.4.2"
 async-trait = "0.1"
 async_zip = { version = "0.0.17", features = ["deflate", "deflate64"] }
+bincode = "1.2.1"
```

Comment:
@mrnugget `bincode` doesn't have a good story around versioning - you can never change the struct definition that is serialized with it to add/remove fields for example. For something persisted to disk we should use json/bson or something that has a well-defined structure so it's harder to accidentally break from version to version.

Reply:
Hehe, this was just temporary, we're not going to use `bincode` right now

Reply:
Phew!